CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ compatible with the updates.
 
 ### Added
 
+- Added support to custom layer channels in U-Net.
```

Comment:
This has nothing to do with the ticket in question... This change really should be it's own ticket and PR... Can you remove this (and any accompanying code) from this PR, and add into this the addition of mypy / updated tests?

Reply:
it's gone now, it was due to the fact that this branch was in advance of `main` and at that time there was another branch not merged into `main` yet. ^^